tlatools/org.lamport.tlatools/src/tla2sany/xml/XMLExporter.java

```diff
@@ -161,7 +155,8 @@ public static final void main(String[] args) throws XMLExportingException {
       if (!offline_mode) { //skip validation in offline mode
         try {
           SchemaFactory factory = SchemaFactory.newInstance(W3C_XML_SCHEMA);
-          Schema schema = factory.newSchema(new URL(TLA_SCHEMA));
+          URL schemaFile = XMLExporter.class.getResource("sany.xsd");
```

Comment:
Catch and report an error if `sany.xsd` is mistakenly missing from `tla2tools.jar`?